package.json

```diff
@@ -160,5 +160,8 @@
     "webpack-cli": "^4.9.0",
     "xml2js": "0.5.0"
   },
+  "overrides": {
```

Comment:
This is the vuln fix that @jennyhliu  noted yesterday?

Reply:
No. I added semver to audit-ci.json, and created a ticket

Reply:
overrides Lerna's semver version and I ad-hoc tested it to make sure it doesn't interfere with Lerna's functioning